### DISCUSSION TITLE
SBFspot + Odroid + HA support

### DISCUSSION THREAD

**Discussion by Digifreakske** [SBFspot/SBFspot#566] [Ideas]
Hi,

Are there any plans to support Odroid systems + HA?
It works great on Pi's but with the chip shortage i switched to Odroid's and the installer checks on PI version.

Regards,
Frederik

**Comment by sbf-** [comment]
Unfortunately I don't own an Odroid (yet) so it's not possible to support the sbfspot-config installer atm
But you can always follow the manual compilation/installation route as described in the wiki for Rpi
